hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -402,10 +403,10 @@ private void markBlocksForDeletionSchemaV1(
       try (BatchOperation batch = containerDB.getStore().getBatchHandler()
           .initBatchOperation()) {
         for (Long blkLong : delTX.getLocalIDList()) {
-          String blk = containerData.blockKey(blkLong);
+          String blk = containerData.getBlockMetaKey(blkLong);
```

Comment:
nit: why is it `BlockMetaKey` and not just `BlockKey`?

Reply:
I think the "meta" here is to indicate the key from Rocksdb, so that "MetaKey" can be used as common suffix for distinction.

Reply:
Hi @symious , thanks for improving the readability of the code.  
I would recommend to use getBlockKey instead of getBlockMetaKey here since the function will return the key's value.  Meta is a word that different people may have different understandings. It will cause confusing.

Reply:
Ok, removed the "meta" from function names, please have a look.